db/migrate/20201113114332_add_uuid_to_course.rb

```diff
@@ -0,0 +1,13 @@
+class AddUuidToCourse < ActiveRecord::Migration[6.0]
+  def up
+    enable_extension "uuid-ossp" unless extension_enabled?("uuid-ossp")
+
+    add_column :course, :uuid, :uuid, default: "uuid_generate_v4()", null: false
```

Comment:
I think this will block read and write on the table. Can we check how long it takes to run? It'll probably be ok as the table isn't that big but we should check.

Reply:
== 20201113114332 AddUuidToCourse: migrating ==================================
-- add_column(:course, :uuid, :uuid, {:default=>"uuid_generate_v4()", :null=>false})
-> 1.2944s
-- add_index(:course, :uuid, {:unique=>true, :name=>"index_courses_unique_uuid"})
-> 0.0571s
== 20201113114332 AddUuidToCourse: migrated (1.3526s) =========================


With 56389 courses

Reply:
👍